content/en/observability_pipelines/destinations/http_client.md

```diff
@@ -0,0 +1,39 @@
+---
+title: HTTP Client Destination
+disable_toc: false
+---
+
+## Overview
+
+Use Observability Pipelines' HTTP Client destination to send logs to an HTTP client, such as a logging platform or SIEM.
+
+## Set up destination
+
+Set up the New Relic destination and its environment variables when you [set up a pipeline][1]. The information below is configured in the pipelines UI.
```

Comment:
is this specifically to New Relic? i think this might have been a copy pasta.

Reply:
Totally copy pasta'd it 🙈...thanks for catching!

```suggestion
Set up the HTTP Client destination and its environment variables when you [set up a pipeline][1]. The information below is configured in the pipelines UI.
```